content/docs/glossary/transactions.mdx

```diff
@@ -42,10 +42,39 @@ The memo contains optional extra information. It is the responsibility of the cl
+
+#### Time Bounds
 The optional UNIX timestamp (in seconds), determined by ledger time, of a lower and upper bound of when this transaction will be valid. If a transaction is submitted too early or too late, it will fail to make it into the transaction set. `maxTime` equal `0` means that it's not set. _We highly advise for all transactions to use time bounds, and many SDKs enforce their usage._ If a transaction doesn't make it into the transaction set, it is kept around in memory in order to be added to the next transaction set on a best-effort basis. Because of this behavior, we highly advise that all transactions are created with time bounds in order to invalidate transactions after a certain amount of time, especially if you plan to resubmit your transaction at a later time.
 
+#### Ledger Bounds
+These are like [Time Bounds](#time-bounds), except they apply to ledger numbers. With them set, a transaction will only be valid for ledger numbers that fall into the range you set. The lower bound is inclusive while the upper bound is not. If you set the upper bound to zero, this indicates that there is no upper bound.
+
+#### Minimum Sequence Number
+If this precondition is set, the transaction will only be valid when its source account's sequence number (call it `S`) is large enough. Specifically, it's valid when `S` satisfies `minSeqNum <= S < tx.seqNum`. If this precondition is omitted, the default behavior applies: the transaction's sequence number must be exactly one greater than the account's sequence number.
+
+Note that after a transaction is executed, the account will always raise its sequence number to the transaction's sequence number.
+
+#### Minimum Sequence Age
+With this precondition, the transaction is only valid after an account's last sequence number change exceeds a particular duration (expressed in seconds).
+
+With a minimum [time bound](#time-bounds), the transaction becomes valid after a particular amount of time passes unconditionally. With this precondition, the passage of time applies only to the transaction's source account: you can allow validity after an *account* has waited long enough since its last sequence number change.
+
+#### Minimum Sequence Ledger Gap
+With this precondition, the transaction is only valid after the current network ledger number meets (or exceeds) a particular gap relative to the ledger corresponding to the source account's last sequence number change.
+
+This is similar to the [minimum sequence age](#minimum-sequence-age), except it's expressed as a quanity of ledgers rather than a duration of time.
+
+#### Extra Signers
+You can require that the transaction have up to two additional signatures beyond the ones required for transaction's authorization in general. There must additional, one-off signatures corresponding to every extra signer in this list, even if the signature is not otherwise required by the sourceAccount or operations.
```

Comment:
Subtle changes regarding signers vs signatures:
```suggestion
You can require that the transaction have up to two additional signers beyond the ones required for transaction's authorization in general. For each extra signer there must be a signature corresponding to the signer, even if the signature would not otherwise be required by the transaction's source account or operations.
```

Reply:
By adding extra signers, you require extra signatures on the transaction. To say you can require extra signers doesn't really make sense.

Perhaps this is a better reword?
```suggestion
A transaction can specify up to two extra signers as a precondition, meaning it must have corresponding additional signatures, even if those signatures would not otherwise be required to authorize the transaction (i.e. for its source account or operations).
```

Reply:
My edit says there has to be a signature corresponding to the signer, not that it requires extra signers. An extra signer may require the same signature as an operation already does, so strictly speaking an extra signer doesn't require an extra signature, or even any additional signatures, there just has to be corresponding signatures to signers.

Your edit is fine too, if you remove the `additional` word:
```suggestion
A transaction can specify up to two extra signers as a precondition, meaning it must have signatures that correspond to those extra signers, even if those signatures would not otherwise be required to authorize the transaction (i.e. for its source account or operations).
```

Reply:
It does say that, though, doesn't it? :smile: 

> You can require that the transaction have up to two additional signers 

Good point regarding redundant signatures, though.